File: projects/m1/028-prime-factors/solutions/MatteoFaienza/prime-factors.py
user_number = int(input('Enter a integer number (2 or greater): ')) 

factor = 2

if user_number < 2 :
    print('Error, enter a integer number (2 or greater)')
    
else :
    print('The prime factorization of',user_number,'is:')

while user_number >= 2 :

    if user_number % factor == 0 :
        user_number = user_number // factor
        print(factor) 
    
    else :
        factor += 1

File: projects/m1/028-prime-factors/solutions/alessiacaporaso/prime-factors.js
// reads an integer from the user's enter
let userNumber = parseInt(prompt('Enter an integer (2 or greater)'))
let factor = 2;
// display error if the user enters a value less than 2
if(userNumber < factor) {
  alert('Error! Can\'t insert a number less than 2 ')
} else {
// your program should display the prime numbers that can be multiplied together to compute n
console.log(`The prime factors of ${userNumber} are:`)
while(userNumber >= factor){
  if(userNumber % factor === 0){
    userNumber /= factor
  } else {
    factor += 1
  }
   console.log(factor)
  }
}